cmd/list_workflows.go

```diff
@@ -26,32 +26,32 @@ var listWorkflowsCmd = &cobra.Command{
 
 		fileFlag, err := flags.GetString("file")
 		if err != nil {
-			u.PrintMessageInColor(fmt.Sprintf("Error getting the 'file' flag: %v", err), theme.Colors.Error)
+			u.PrintErrorMarkdownAndExit("Invalid Usage", fmt.Errorf("Error getting the `file` flag: %v", err), "")
 			return
 		}
 
 		formatFlag, err := flags.GetString("format")
 		if err != nil {
-			u.PrintMessageInColor(fmt.Sprintf("Error getting the 'format' flag: %v", err), theme.Colors.Error)
+			u.PrintErrorMarkdownAndExit("Invalid Usage", fmt.Errorf("Error getting the `format` flag: %v", err), "")
 			return
 		}
 
 		delimiterFlag, err := flags.GetString("delimiter")
 		if err != nil {
-			u.PrintMessageInColor(fmt.Sprintf("Error getting the 'delimiter' flag: %v", err), theme.Colors.Error)
+			u.PrintErrorMarkdownAndExit("Invalid Usage", fmt.Errorf("Error getting the `delimiter` flag: %v", err), "")
 			return
 		}
 
 		configAndStacksInfo := schema.ConfigAndStacksInfo{}
 		atmosConfig, err := config.InitCliConfig(configAndStacksInfo, true)
 		if err != nil {
-			u.PrintMessageInColor(fmt.Sprintf("Error initializing CLI config: %v", err), theme.Colors.Error)
+			u.PrintErrorMarkdownAndExit("Error initializing CLI config", err, "")
```

Comment:
Looks like we didn't used to exit

Reply:
Actually we had the return statement bellow. That in turn exits the cli with exit code 0. So Now it would exit with exit code 1 which is correct as there was error. 😅